hivemq-edge/src/main/java/com/hivemq/configuration/entity/HiveMQConfigEntity.java

```diff
@@ -44,6 +45,11 @@
 @SuppressWarnings({"FieldMayBeFinal", "FieldCanBeLocal"})
 public class HiveMQConfigEntity {
 
+    public static final int CURRENT_CONFIG_VERSION = 1;
```

Comment:
We need something to remind us to increase the number when something actually changes.
Like a breaking test.
Also: What happens with the configs contributed by adapters?

Reply:
I added an integration test for the json schema that is generated by each adapter